composer.json

```diff
@@ -20,6 +20,7 @@
   },
   "require": {
     "php": ">=5.3.3",
-    "apache/log4php": "2.3.0"
+    "apache/log4php": "2.3.0",
```

Comment:
@jacobkiers @johanwilfer how would that work with an "out of the box" installation? If one does not specify a PSR3 logger with the new option, [there's a default implementation](https://github.com/marcelog/PAMI/pull/83/files#diff-619e399f4b58646d77fb4f82900c73dbR445) so backwards compatibility is mantained and log4php is used. If it defined as a "suggested" package, we would have to declare all PSR3 implementors, just to be fair with them. I think we should declare it as required until (let's say) v2.0.0. 

Thoughts?